packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -280,6 +278,9 @@ export default defineComponent({
 			useUsersStore,
 			useWorkflowsStore,
 		),
+		isReadOnlyRoute() {
```

Comment:
Same here (and other components, I won't comment everywhere). The readonly state should be passed down as prop.. but I don't expect any bugs here related to this or the other ones